container/container.go

```diff
@@ -56,6 +56,10 @@ func StopContainer(client DockerClient, containerId string, timeout uint) error
 		return err
 	}
 
+	if cntr.State.Status == "exited" {
```

Comment:
Are you sure "exited" is the status string in this case? I'm not sure it's not, but I remember "Exited" somewhere